tests/unit/definitions_test.c

```diff
@@ -0,0 +1,38 @@
+#include <platform.h>
+#include <definitions.h>
+
+#include <test.h>
+
+static void test_metric_prefixes(void)
+{
+    assert_int_equal(KIBIBYTE(0), 0);
+    assert_int_equal(KIBIBYTE(4), 4096);
+    assert_int_equal(KIBIBYTE(8), 8192);
+    assert_int_equal(KIBIBYTE(16), 16384);
+
+    assert_int_equal(MEBIBYTE(1), 1048576);
+    assert_int_equal(MEBIBYTE(5), 5242880);
+    assert_int_equal(MEBIBYTE(9), 9437184);
+    assert_int_equal(MEBIBYTE(17), 17825790);
+
+    assert_int_equal(GIBIBYTE(2), 2147484000);
+    assert_int_equal(GIBIBYTE(6), 6442451000);
+    assert_int_equal(GIBIBYTE(10), 10737420000);
+    assert_int_equal(GIBIBYTE(18), 19327350000);
+
+    assert_int_equal(TEBIBYTE(3), 3298535000000);
+    assert_int_equal(TEBIBYTE(7), 7696581000000);
+    assert_int_equal(TEBIBYTE(11), 12094630000000);
+    assert_int_equal(TEBIBYTE(19), 20890720000000);
+}
+
+int main()
+{
+    PRINT_TEST_BANNER();
+    const UnitTest tests[] =
+    {
+        unit_test(test_metric_prefixes())
```

Comment:
```suggestion
        unit_test(test_metric_prefixes)
```

Give it the function pointer, don't call the function.